websocket/polygon.go

```diff
@@ -329,16 +329,15 @@ func (c *Client) write() error {
 			if err := c.conn.WriteMessage(websocket.TextMessage, msg); err != nil {
 				return fmt.Errorf("failed to send message: %w", err)
 			}
-		default:
```

Comment:
i think these defaults could cause the write and process threads to be cpu hungry. deleting so they can block until data is available.